web/src/app/admin/admin-outgoing-logs/OutgoingLogsControls.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react'
+import { Grid, Typography, IconButton } from '@mui/material'
+import makeStyles from '@mui/styles/makeStyles'
+import RestartAltIcon from '@mui/icons-material/RestartAlt'
+import { theme } from '../../mui'
+import { ISODateTimePicker } from '../../util/ISOPickers'
+import { useResetURLParams, useURLParam } from '../../actions'
+import Search from '../../util/Search'
+import { LOAD_AMOUNT } from './OutgoingLogsList'
+
+interface Props {
+  totalCount: number
+}
+
+const useStyles = makeStyles<typeof theme>({
+  filterContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  resetButton: {
+    height: 'min-content',
+    alignSelf: 'center',
+  },
+})
+
+export default function OutgoingLogsControls(p: Props): JSX.Element {
+  const classes = useStyles()
+
+  const [start, setStart] = useURLParam<string>('start', '')
+  const [end, setEnd] = useURLParam<string>('end', '')
+  const [key, setKey] = useState(0)
+  const resetDateRange = useResetURLParams('start', 'end')
+  const [limit] = useURLParam<string>('limit', '1')
+  const _limit = parseInt(limit, 10)
+
+  const resetFilters = (): void => {
+    resetDateRange()
+    setKey(key + 1)
```

Comment:
What's the purpose of `setKey` here? Forcing remount due to a bug?

Reply:
Yep, this is to get around a bug. The ISODateTimePicker doesn't seem to update when you change it's value. This was a temporary fix to allow changing the ISODateTimePicker's value as if it were a controlled component by just completely re-rendering it. I tried seeing if I could find where the issue is in the ISO picker but it wasn't an easy fix so I added this for now.

Reply:
I see that seems reasonable. Throw a comment in with a brief explanation for why it's necessary so it doesn't get removed later

Reply:
Added 👍